Makefile

```diff
@@ -16,8 +16,22 @@
 verify-boilerplate:
 	@./hack/verify-boilerplate.py --rootdir=$(CURDIR) --boilerplate-dir=hack/boilerplate && echo "Boilerplate verification passed."
 
+.PHONY: prowgen
+prowgen:
+	cd ./config/prowgen/ && go run . --branch=* -o $(CURDIR)/config/jobs/cert-manager/cert-manager/
```

Comment:
It'd be nice to have the branch configurable as I think a common pattern when making changes will be to apply them to master branch only

Reply:
I think that all branches should always be in sync with the code, so I'm not sure why you would want to update once branch only. If needed for local development only, you can run the command yourself or undo the changes in git.